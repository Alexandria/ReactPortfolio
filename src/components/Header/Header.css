.jumbotron {
  background-color: #1f2833;
  background-image: linear-gradient(to bottom right, #45a29e, #1f2833);
  padding-top: 18%;
  padding-bottom: 18%;
}
.display-4 {
  color: aliceblue;
  font-family: "Anton", sans-serif;
  font-size: 100px;
}
.lead {
  color: aliceblue;
  font-size: 45px;
  font-family: "Anton", sans-serif;
}
