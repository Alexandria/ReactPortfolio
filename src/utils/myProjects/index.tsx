import { Project } from "../../components/ProjectCard/ProjectCard";
import typescript from "../../assests/icons/typescript.png";
import Meeplely from "../../assests/images/Meeplely.png";
import nodeJs from "../../assests/icons/nodeJs.png";
import react from "../../assests/icons/react.png";
import postgres from "../../assests/icons/postgres.png";
import python from "../../assests/icons/python.png";
import phpmyadmin from "../../assests/icons/phpmyadmin.png";
import insomnia from "../../assests/icons/insomnia.png";
import django from "../../assests/icons/django.jpg";

export const myProjects: Project[] = [
  {
    title: "Meeplely",
    img: Meeplely,
    description:
      "Full stack web application that will help you manage your board game collection.",
    toolIcons: [typescript, nodeJs, react, postgres, insomnia]
  },
  {
    title: "Justice for Barb Blog",
    img: Meeplely,
    description:
      "Full stack web application that will help you manage your board game collection.",
    toolIcons: [django, python, phpmyadmin]
  }
];

export const project: Project = {
  title: "Meeplely",
  img: Meeplely,
  description:
    "Full stack web application that will help you manage your board game collection.",
  toolIcons: [typescript, nodeJs, react, postgres]
};
