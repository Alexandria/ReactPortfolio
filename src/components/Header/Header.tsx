import React from "react";
import "./Header.css";

const Header: React.FC = () => {
  return (
    <div className="jumbotron ">
      <div className="container">
        <h1 className="display-4">Lex Sanders</h1>
        <p className="lead">Software Developer</p>
      </div>
    </div>
  );
};

export default Header;
