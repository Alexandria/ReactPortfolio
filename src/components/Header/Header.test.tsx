import React from "react";
import Header from "./Header";
import { shallowRender } from "../../utils/shallowRender";

describe("Header", () => {
  it("renders expected structure", () => {
    const received = shallowRender(<Header />);
    expect(received).toMatchSnapshot();
  });
});
