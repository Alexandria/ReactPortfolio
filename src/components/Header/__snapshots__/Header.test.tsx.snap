// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders expected structure 1`] = `
<div
  className="jumbotron "
>
  <div
    className="container"
  >
    <h1
      className="display-4"
    >
      Lex Sanders
    </h1>
    <p
      className="lead"
    >
      Software Developer
    </p>
  </div>
</div>
`;
